nc-games-fe/src/Components/Review-Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { fetchReviewComments } from "./utils";
+
+const ReviewComments = ({ reviewId }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasComments, setHasComments] = useState(false);
+
+  useEffect(() => {
+    fetchReviewComments(reviewId)
+      .then((data) => {
+        setComments(data.comments);
+        setIsLoading(false);
+        setHasComments(data.comments.length > 0);
+      });
+  }, [reviewId]);
+
+  if (isLoading) return <p>Loading comments...</p>;
+
+  if (!hasComments) {
+    return <p>No comments here yet!</p>
+  }
```

Comment:
Im a big fan of both of these being here 🙌, but do you require a dedicated state to check if there are comments? or is there another way to check for this?